cmd/lk/sip.go

```diff
@@ -369,6 +369,10 @@ var (
 									Usage: "timeout for the call to dial",
 									Value: 80 * time.Second,
 								},
+								&cli.StringSliceFlag{
+									Name:  "headers",
```

Comment:
```suggestion
									Name:  "header",
```
Because it is defined as an array, so one must repeat it: `--header K1:V1 --header K2:V2`.